### PR TITLE
Mac x86 runthrough

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: cloud-native/README.md [hunk 1]
```diff
@@ -559,25 +563,38 @@ $ minikube addons enable registry
...
-_Note: As the message indicates, be sure you use the correct port instead of 5000_
-
-We can now build the image directly using `minikube image build`:
-On Linux and macOS:
+_Note: As the message indicates, be sure you use the correct port instead of 5000_.
+On Linux and macOS export a variable with the registry with:
 
 ```console
-minikube image build -t $(minikube ip):<port>/nodeserver:1.0.0 --file Dockerfile-run .
+export REGISTRY=$(minikube ip):<port>minikube addons enable registry
```
Comment (pacostas) [inline cloud-native/README.md:570]: (super minor) Changing it to something like MINIKUBE_REGISTRY might also be a good name.

File: cloud-native/README.md [hunk 2]
```diff
@@ -572,6 +572,18 @@ export REGISTRY=$(minikube ip):<port>minikube addons enable registry
...
+On Windows export a variable with the registry with by first running 
+
+```
+minikube ip
+```
+
+to get the ip of the registry and then exporting
+
+```
+set REGISTRY=<ip from minikube ip command above>:5000
```
Comment (pacostas) [inline cloud-native/README.md:584]:
```suggestion
set REGISTRY=<ip from minikube ip command above>:<port>
```

Comment (mhdawson) [reply cloud-native/README.md:584]: @pacostas I found that when I enabled minikube on my windows it did not have an alternate port, and so it was just 5000?

File: cloud-native/README.md (hunk 1 above)
Comment (mhdawson) [reply cloud-native/README.md]: @pacostas updated.

File: cloud-native/README.md (hunk 2 above)
Comment (mhdawson) [reply cloud-native/README.md]: expanded to try to cover both cases where it recommends a different port and when it does not.

Comment (pacostas) [reply cloud-native/README.md]: > expanded to try to cover both cases where it recommends a different port and when it does not.

I agree on the extra instructions about the port.

**Comment by pacostas** [comment]
LGTM

**Pull request merged**